packages/sources/src/components/alert/alert.scss

```diff
@@ -23,13 +21,47 @@ $alert-danger-bg: $state-danger-bg;
 $alert-danger-text: $state-danger-text;
 $alert-danger-border: $state-danger-border;
 
-@mixin alert-variant($background, $border, $text-color) {
+@mixin alert_children() {
+  > .dso-rich-content {
+    *:first-child {
+      margin-top: 0;
+    }
+
+    *:last-child {
+      margin-bottom: 0;
+    }
+
+    #{$headings} {
```

Comment:
Hier ook nog even over brainen

Reply:
Op zich best geinig, maar ik kan me voorstellen dat we dit hier gewoon willen uittypen als `h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6`

Wat vind jij?

Reply:
We laten het nu maar even, de variable bestaat al en het is gewoon 'n verhuizing van styling: https://github.com/dso-toolkit/dso-toolkit/pull/1171/files#diff-11f75f85c5f594b1ab95874f165cda74821133393356d449474228da79f2d72aL80